src/dev-theme.css

```diff
@@ -59,6 +65,10 @@ ul > li {
   color: var(--rgba-secondary-1-1);
 }
 
+.themed-text{
+  font-family: 'Andika';
+}
```

Comment:
Can you add a font-size of like 1.05em or similar, to increase read-ability (without making too big for phone-sized screens)?